### ISSUE TITLE
IOS App Issue

### ISSUE THREAD

**Issue by hamzaamir749** [stevdza-san/RoomKMP_Demo#1]
![Screenshot 2024-10-05 at 17 17 28](https://github.com/user-attachments/assets/cf08ea11-0dd7-49a8-a745-f11b797adde0)

**Comment by stevdza-san** [comment]
Have you cloned my repo?
Try clearing your IDE/Build cache?

**Comment by hamzaamir749** [comment]
Yes cleaned and I didn't clone your repo I followed everything that
mentioned in the video and also compare my things with your repo.

**Comment by stevdza-san** [comment]
Also try executing this command in your terminal to try and fix issues with kotlin build: ```jps | grep -E 'KotlinCompileDaemon' | awk '{print $1}' | xargs kill -9 || true```

**Comment by hamzaamir749** [comment]
Yes sure I will also clone your repo and try to run that as well. I will
let you know thank you for your response.

**Comment by hamzaamir749** [comment]
same error i got after running command and i also clone your repo got same
error *copyRoomSchemasToAndroidTestAssetsDebugAndroidTest.*

**Comment by oscarb** [comment]
I was able to fix the same kind of error in another project by adding this to my `gradle.properties` file:

```
org.gradle.unsafe.configuration-cache=true
```

Maybe that works for this project as well?
